mailpoet/lib/API/JSON/v1/Services.php

```diff
@@ -211,6 +211,11 @@ public function recheckKeys() {
     $this->mssWorker->checkKey();
     $this->premiumWorker->init();
     $this->premiumWorker->checkKey();
+    // continue sending when it is paused
+    $mailerLog = MailerLog::getMailerLog();
+    if (isset($mailerLog['status']) && $mailerLog['status'] === MailerLog::STATUS_PAUSED) {
+      MailerLog::resumeSending();
+    }
```

Comment:
Is it safe to always resume the sending? Shouldn't we resume the sending based on the status returned by those checks?

Reply:
Good point! I will change it.